### DISCUSSION TITLE
Radio & Checkbox widgets

### DISCUSSION THREAD

**Discussion by KellerKev** [emmett-framework/emmett#328] [Q&A] [answered]
Does the Emmett form feature Radio and Checkbox widgets?

**Comment by gi0baro** [comment, marked as answer]
Yes, they are automatically used based on the type of your fields and their validation.

Code references:
- select: https://github.com/emmett-framework/emmett/blob/master/emmett/forms.py#L508#L524
- multiple: https://github.com/emmett-framework/emmett/blob/master/emmett/forms.py#L527#L540

You can also use your custom widgets, see https://emmett.sh/docs/2.2.x/orm/models#widgets

**Comment by KellerKev** [comment]
Excellent, thank you!
